reusable_learning_app/lib/screens/Content/settings_body.dart

```diff
@@ -13,18 +14,261 @@ class SettingsBody extends StatefulWidget implements NavItem   {
 }
 
 class SettingsBodyState extends State<SettingsBody> {
```

Comment:
Please, make sure you follow the overall design of the app. Shadows are too visible and blurry. Make them similar as we have in the prototype.

<div style="display:flex; flex-direction: row; flex-">
  <img width="320" alt="image" src="https://user-images.githubusercontent.com/62881333/174034717-7dcf5353-e21e-4801-a6da-886a2330c51d.png">
  <img width="301" alt="image" src="https://user-images.githubusercontent.com/62881333/174035294-9e402627-e94c-4d33-b2a0-b782559f6add.png">
</div>

Reply:
Done. Changes commited